napari/layers/intensity_mixin.py

```diff
@@ -40,6 +40,9 @@ def reset_contrast_limits(self: 'Image', mode=None):
         mode = mode or self._autoscale_source
         self.contrast_limits = self._calc_data_range(mode)
 
+    def _reset_contrast_limits_signal(self):
```

Comment:
can't immediately see where this is used?

Reply:
Nowhere. Old workaround for signal bind problem.

Reply:
Thanks, can you take another look through this PR and make sure that all the temporary stuff is gone?  Would be nice to get this in